src/text/text.rs

```diff
@@ -225,6 +225,19 @@ where
     }
 }
 
+impl std::fmt::Display for Text<'_> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let vec_len = self.lines.len();
+        for (i, line) in self.lines.iter().enumerate() {
+            write!(f, "{line}")?;
+            if (i + 1) != vec_len {
+                writeln!(f)?
+            }
+        }
```

Comment:
One alternate way of doing this is to use https://docs.rs/itertools/latest/itertools/trait.Itertools.html#method.with_position
```suggestion
        for (position, line) in self.lines.iter().with_positions() {
            if position == Position::Last {
                write(f, "{line}")?;
            } else {
                writeln(f, "{line}")?;
            }
        }
```

Reply:
That works too, it just depends on whether we want an ending `\n`. I don't know what makes the most sense.

Reply:
We likely don't - think of the single line case - that should render "Hello", two lines "Hello\nWorld".